### DISCUSSION TITLE
Docs about security

### DISCUSSION THREAD

**Discussion by killjin** [livewire/livewire#8501] [7. Bugs (archived)] [answered]
### Livewire version

3.5

### Laravel version

v11

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

https://livewire.laravel.com/docs/security#using-model-properties

If the client simulates a snapshot, it can also change the component's postid props. Is this also unsafe?

### Repository or Wirebox URL

https://livewire.laravel.com/docs/security

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
@killjin Have a look at this section on snapshot checksums. The checksum is what stops people tampering with or creating their own snapshots. It's created using your app key, so no one else should be able to generate it https://livewire.laravel.com/docs/security#snapshot-checksums
